draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
       </section>
     </section>
 
+    <section title="SDP Identity Attribute" anchor="sec.sdp-id-attr">
+      <t>
+	    The SDP 'identity' attribute is a session-level attribute that is used
+	    by an endpoint to convey its identity assertion to its peer. The identity
+        assertion value is encoded as a <xref target="RFC4648">Base-64 string</xref>
+      </t>
+      <t>
+	    The procedures in this section are based on the assumption that the identity assertion 
+        of an endpoint is bound to the fingerprints of the endpoint. This does not preclude the 
+        definition of alternative means of binding an assertion to the identity, but such means are 
+        outside the scope of this specification.
+      </t>
+      <t>
+	    The semantics of multiple 'identity' attributes within an offer or answer are undefined.
+	    Implementations SHOULD only include a single identity attribute in an
+	    offer or answer and relying parties MAY elect to ignore all but the first
+	    identity attribute. However, a single identity attribute can convey multiple 
+	    identity assertions.
+      </t>
+      <figure>
+      <artwork type="inline"><![CDATA[
+
+Name: identity
+
+Value: identity-assertion
+
+Usage Level: session
+
+Charset Dependent: no
+
+Default Value: N/A
+
+Syntax:
+
+  identity-assertion        = identity-assertion-value 
+                              *(SP identity-extension)
+  identity-assertion-value  = base64
+  base64                    = 1*(ALPHA / DIGIT / "+" / "/" / "=" )
+  identity-extension        = extension-name [ "=" extension-value ]
+  extension-name            = token
+  extension-value           = 1*(%x01-09 / %x0b-0c / %x0e-3a / %x3c-ff)
+                              ; byte-string from [RFC4566] omitting ";"
```

Comment:
Why is the ' omitted ?

Reply:
Yeah, we split on space.  semicolon should be fine.

Reply:
I forgot to remove the semicolon restriction. Will do that in the next commit.